test/integration/patients/patient/patient-flow.js

```diff
@@ -847,6 +847,172 @@ context('patient flow page', function() {
       .should('not.exist');
   });
 
+  specify('flow with only work:team:manage permission', function() {
```

Comment:
In this test, I set up a patient dashboard with four flows:

1. `'Owned by current clinician’s team'`
2. `'Owned by another team'`
3. `'Owned by team member'`
4. `'Owned by non team member'`

Then I went through each flow in the patient dashboard's list, click through to the `/flow:id` page, and test that the user can/can't make edits to the flow (state and/or owner in header section of the page).

It's similar to the patient flow sidebar test, but tests the header section instead of the sidebar.